WooCommerce/Classes/ViewRelated/Products/UpdateInventory/UpdateProductInventoryView.swift

```diff
@@ -68,24 +68,33 @@ struct UpdateProductInventoryView: View {
                         Spacer()
 
                         Group {
-                            Button(Localization.updateQuantityButtonTitle) {}
-                                .buttonStyle(PrimaryButtonStyle())
-                                .renderedIf(viewModel.updateQuantityButtonMode == .customQuantity)
-
-                            Button(Localization.increaseStockOnceButtonTitle) {}
-                                .buttonStyle(PrimaryButtonStyle())
-                                .renderedIf(viewModel.updateQuantityButtonMode == .increaseOnce)
+                            Button(Localization.updateQuantityButtonTitle) {
+                                Task { @MainActor in
+                                    await viewModel.onTapUpdateStockQuantity()
+                                    dismiss()
+                                }
+                            }
+
```

Comment:
It might make the `renderedIf` clearer about its target, if we remove this blank line.

Reply:
Good point! Done in ce180b2aff880043b567d1e263abe1c65bc6c9d0